src/tls/cockpit-certificate-helper.in

```diff
@@ -56,6 +56,17 @@ cmd_ipa_request() {
     install_key 10-ipa.key
 }
 
+cmd_ipa_cleanup() {
+        # clean up keytab
+        if [ -e "${KEYTAB}" ]; then
+            ipa-rmkeytab -k "${KEYTAB}" -p "${SERVICE}"
+        fi
+
+        # clean up certificate
+        ipa-getcert stop-tracking -f 10-ipa.cert -k 10-ipa.key
+        rm -f "${COCKPIT_WS_CERTS_D}"/10-ipa.{cert,key}
```

Comment:
Bashism, rather spell this out. This may possibly be the reason for the Debian/Ubuntu test failure? (I didn't look closely)

Reply:
after trying about 10 other ways to avoid repeating the variable expansion, i couldn't find anything that was truly nicer...